src/rules/add-metadata-to-token.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @title User metadata for Cumul.io
+ * @overview Add Cumul.io user metaadata to token to be used for Cumul.io dashboard filtering
+ * @gallery true
+ * @category marketplace
+ */
```

Comment:
One more thing ... There needs to be some text here as an on-page description. We can use what you have at the top of the install guide for now and we'll link to the marketplace listing once it goes live.

> This integration simplifies the process of making full use of integrated Cumul.io dashboards' multi tenant features by using Auth0 as its authentication layer. The integration will allow you to set up and use user information in Auth0 to filter and structure your Cumul.io dashboards.